.github/workflows/detect-breaking-changes.yml

```diff
@@ -6,9 +6,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v3.8.1
```

Comment:
Why the hard version lock here? I'm not sure if locking to a patch version is beneficial to us here.

Reply:
Removed the pinning